### PR TITLE
Add verbaige to define a Jenkinsfile

### PR DESCRIPTION
**Pull request by jamesrgregg**
- Add some verbaige to define a Jenkinsfile 
- Provide an example of a very basic Jenkinsfile as a high level overview

Signed-off-by: James Gregg <james.r.gregg@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,37 @@
 ## Background
 
 * What is a Jenkinsfile?
+  * A Jenkinsfile is a text file that contains the definition of a Jenkins Pipeline and is checked into source control.
```
Comment (ernestojeda) [inline README.md]: A Jenkinsfile is a text file that contains the build definition (all the components required to build and deploy the source code of your project) of a Jenkins Pipeline and is checked into source control. ?? Maybe mention a Jenkinsfile is written in Groovy.

**Review by ernestojeda — CHANGES_REQUESTED** [review]
I added a comment in the `What is a Jenkinsfile?` section

File: README.md (hunk 1 above)
Comment (jamesrgregg) [reply README.md]: lol.. I had suggested you add that and you ignored me.

Comment (jamesrgregg) [reply README.md]: @ernestojeda I have added this as you suggested.

**Review by ernestojeda — APPROVED** [review]
👍

**Pull request merged**